osu.Desktop/osu.Desktop.csproj

```diff
@@ -26,7 +26,7 @@
   <ItemGroup Label="Package References">
     <PackageReference Include="System.IO.Packaging" Version="8.0.1" />
     <PackageReference Include="DiscordRichPresence" Version="1.2.1.24" />
-    <PackageReference Include="Velopack" Version="0.0.630-g9c52e40" />
+    <PackageReference Include="Velopack" Version="0.0.869" />
```

Comment:
This bump makes me ***INCREDIBLY*** nervous, to the point where I'm hesitant approving this PR without doing an installer check on all platforms even if everything else looks fine (and it does). Should I be doing one?

Reply:
Yeah, the changelog looked tame enough that I was just thinking "let's get off the pre-releases we're currently on".

Reply:
The 755 build (which is used by osu-deploy), is an interim build that already contains the appimage change + revert. So there should be no issue.

Reply:
On a brief check on linux with `osu-deploy` it seems like things work, so I'm gonna cross fingers and hope for the best...